terminal.md

```diff
@@ -0,0 +1,12 @@
+### No 5
+- cd ../
+- cd ../../
+- cd ../work/project_to_delegate
+- cd ../../home
+
+### No 10
+- rm random.txt
+- rm -rf home
+- cd ../ rm -rf home

```

Comment:
I know you meant to put these two commands on separate lines.

But if you want to execute a one liner, here's a tip: `cd .. && rm -rf home`.

Chain as many commands separated with `&&`

Reply:
yeah, thanks for your feedback and I've done the rest of the homework